libtsuba/src/RDG.cpp

```diff
@@ -451,12 +472,16 @@ tsuba::RDG::DoMake(const katana::Uri& metadata_dir) {
   }
 
   KATANA_LOG_DEBUG(
-      "ReadPartMetadata master sz: {} mirrors sz: {} h2owned sz: {} l2u sz: "
+      "ReadPartMetadata master sz: {} mirrors sz: {} h2on sz: {} h20e sz: {} "
```

Comment:
Why do we print this all the time in debug mode?

Reply:
Good question. I think @tylershunt may have written the first version of this print?